src/components/InvestTab/index.jsx

```diff
@@ -233,7 +233,12 @@ export default class InvestTab extends React.Component {
         procID: null,
       }, () => {
         saveJob(job);
-        cleanupDir(tempDir);
+        fs.unlink(datastackPath, (err) => {
+          if (err) { logger.error(err); }
+          fs.rmdir(tempDir, (e) => {
+            if (e) { logger.error(e); }
+          });
+        });

```

Comment:
Did it make sense to remove this from `utils.js` because it wasn't used anywhere else?

Reply:
It was used several places in tests, and this was the only instance in src. But that `cleanupDir` function was being flaky and had a bug that was intermittently causing tests to error. It didn't really feel like it was worth maintaining our own function.